code/modules/interface/multiContext/multiContext.dm

```diff
@@ -33,6 +33,7 @@
 		var/atom/movable/screen/contextButton/B = new /atom/movable/screen/contextButton
 		B.setup(C, src, target)
 		B.alpha = 0
+		B.dir = src.dir // atmos pipe crafting uses this to show the pipe dir
```

Comment:
you set it here but it never uses it

Reply:
the pipe button icons i added have four directions, so the button image will visually reflect what pipe they will craft if they hit the button and face their current direction

Reply:
removed this part too